draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1944,8 +1942,8 @@ consisting of:
 
 ## SCT Extensions    {#sct_extension_types}
 
-IANA is asked to establish a registry of SCT extensions, named "Signed
-Certificate Timestamp Extension types" initially consisting of:
+IANA is asked to establish a registry of SCT extensions, named "CT Extension
+Types for SCT" initially consisting of:
```

Comment:
There needs to be a comma before "initially".

I think "s/ initially consisting/, that initially consists/" would be better still.